packages/theme-toolkit/src/component-stories-utrecht.tsx

```diff
@@ -1928,6 +1928,27 @@ export const UTRECHT_COMPONENT_STORIES: ComponentStory[] = [
         <SelectOption value="3">Option 3</SelectOption>
       </Select>
     ),
+    detectTokens: {
+      anyOf: [
+        'utrecht.select.background-color',
+        'utrecht.select.image',
```

Comment:
```suggestion
        'utrecht.select.background-image',
```
`image` is niet een bestaande token, dat moet `background-image` zijn